ci/set_port_range.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# See http://mesos.apache.org/documentation/latest/port-mapping-isolator/
+# and mesosphere.util.PortAllocator docs.
+echo "Ephemeral port range before: $(cat /proc/sys/net/ipv4/ip_local_port_range)"
+sysctl -w net.ipv4.ip_local_port_range="60001 61000"
+echo "Ephemeral port range after: $(cat /proc/sys/net/ipv4/ip_local_port_range)"
```

Comment:
why do we need to do that?

Reply:
This relates to https://github.com/mesosphere/usi/blob/master/test-utils/src/main/scala/com/mesosphere/utils/PortAllocator.scala#L11. Alex found that some of our parallel integration tests failed because ports would be assigned twice and thus services wouldn't be able to bind to a certain port. Unfortunately this is the only way to tell Mesos a port range.